src/typechecker.erl

```diff
@@ -4806,8 +4821,7 @@ create_fenv(Specs, Funs) ->
       [ {{Name, NArgs}, type(any)}
         || {function,_, Name, NArgs, _Clauses} <- Funs
       ] ++
-      [ {{Name, NArgs}, lists:map(fun typelib:remove_pos/1,
-                                  absform:normalize_function_type_list(Types))}
+      [ {{Name, NArgs}, absform:normalize_function_type_list(Types)}
```

Comment:
So we keep the positions in fenv but we remove them in tenv. Don't we need them in tenv too for the same reasons?

Reply:
> I wonder where we normalize records from other modules.

I think we can only access them through `gradualizer_db` and there's code like this there: https://github.com/josefs/Gradualizer/blob/5be0f94d0556e8f813963d3f907c51c0da8077f3/src/gradualizer_db.erl#L506-L535